test/addons/openssl-binding/test.js

```diff
@@ -1,6 +1,10 @@
 'use strict';
 
 const common = require('../../common');
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  process.exit(0);
```

Comment:
I'll take another look at this. Just returning does not work in this case but I'll try and dig a little further.